lib/index.ts

```diff
@@ -120,12 +120,14 @@ export function detectPrng(allowInsecure: boolean = false, root?: any): PRNG {
     root = typeof window !== "undefined" ? window : null
   }
 
-  const browserCrypto = root && (root.crypto || root.msCrypto)
+  const webCrypto = root &&
+    (root.crypto || root.msCrypto) ||
+    (typeof crypto !== "undefined" ? crypto : null)
```

Comment:
We could probably just do `typeof crypto` and `typeof msCrypto` instead, but I didn't want to break backward compatibility with the `root` that can currently be passed into this function.